File: C++/Array1/LinearSearch.cpp
#include<iostream>
using namespace std;
int main(){
    int n;
    cout<<"Enter the size of array : ";
    cin>>n;
    int arr[n];
    //input
    for(int i=0;i<=n-1;i++){
        cin>>arr[i];
    }
    int x;
    cout<<"Enter the number you want to find : ";
    cin>>x;
    //search
    bool flag = false; //pehle se hi man lege ki element nhi hai
    for(int i=0;i<=n-1;i++){
        if(arr[i]==x) flag=true;
    }
    if(flag==true) cout<<"Element is present ";
    else cout<<"Element not present ";
}

File: JAVA/linearSearch.java
import java.util.Scanner;

public class linearSearch {
    int a[]=new int[10],key;
    void getdata()
    {
        Scanner sc = new Scanner(System.in);
        for(int i=0;i<10;i++)
        {
            System.out.println("Enter the Number : ");
            a[i]=sc.nextInt();
        }
        System.out.print("Enter the number you want to search : ");
        key = sc.nextInt();
    }
    void Search()
    {
        int flag=0,pos;
        for(int i=0; i<10 && flag == 0 ; i++)
        {
            if(a[i]==key)
            {
                flag = 1;
                pos = i+1;
            }
        }
        if(flag==1) System.out.println("Number Found "+pos);
        else System.out.println("Number not Found");
    }
}
class demo{
    public static void main(String[] args) {
        linearSearch ls = new linearSearch();
        ls=getdata();
        ls=Search();

    }
}
